cms/envs/common.py

```diff
@@ -2925,3 +2925,11 @@ def _should_send_learning_badge_events(settings):
 # .. for now it wil impact country listing in auth flow and user profile.
 # .. eg ['US', 'CA']
 DISABLED_COUNTRIES = []
+
+# .. setting_name: LIBRARY_ENABLED_BLOCKS
+# .. setting_default: ['problem', 'video', 'html', 'drag-and-drop-v2']
+# .. setting_description: List of block types that are ready/enabled to be created/used
+# .. in libraries. Both basic blocks and advanced blocks can be included.
+# .. To see the individual configuration per library go to
+# .. openedx/core/djangoapps/content_libraries/api.py::get_allowed_block_types()
+LIBRARY_ENABLED_BLOCKS = ['problem', 'video', 'html', 'drag-and-drop-v2']
```

Comment:
With all the work you've been doing on the advanced editors, are there any other blocks we can turn on by default yet?

Reply:
You're right, some blocks can already be activated. I will to add them.

Reply:
Updated: https://github.com/openedx/edx-platform/pull/36292/commits/b5d61540a0ae38492d3947c9067776c2dd3f83bc